includes/public/layouts/default.php

```diff
@@ -76,7 +76,15 @@ class="photo photo-wrapup wppr-product-image"/></a>
 									<span><?php echo esc_html( number_format( ( $option['value'] / 10 ), 1 ) ); ?>
 										/10 </span>
 								</div>
-								<ul class="cwpr_clearfix <?php echo $review_object->get_rating_class( $option['value'] ) . apply_filters( 'wppr_option_custom_icon', '' ); ?>">
+								<ul class="cwpr_clearfix
+								<?php
+								if ( function_exists( 'ampforwp_is_amp_endpoint' ) && ampforwp_is_amp_endpoint() || function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
+									echo ' ' . $review_object->get_rating_class( $option['value'] ); // AMP isn't accepting custom icons.
+								} else {
+									echo ' ' . $review_object->get_rating_class( $option['value'] ) . apply_filters( 'wppr_option_custom_icon', '' );
```

Comment:
you should hook to this filter `wppr_option_custom_icon` and return empty instead of doing this the check inside the template. 
Make sure you hook to that filter at the latest time (maybe 99 priority )

Reply:
Sure, will try that. Thanks.

Reply:
@selul Can you explain a little bit more on what needs to be done?

Reply:
ok,  i'm saying that you can build a filter which will hook to `wppr_option_custom_icon` and return empty if we are in the amp endpoint, something like: 
```
add_filter('wppr_option_custom_icon',array($this,'clear_amp_custom_icons'),99);
function clear_amp_custom_icons($old_value){
if ( function_exists( 'ampforwp_is_amp_endpoint' ) && ampforwp_is_amp_endpoint() || function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
return '';
}
return $old_value;
}```
you can add this in the public class, it should be enough. i hope it makes sense.

Reply:
Yep. Thanks.

Reply:
Applied the filter and made changes.